undercloud.yml

```diff
@@ -1,3 +1,48 @@
+- hosts: undercloud
+  tasks:
```

Comment:
you can create separate tasks file for setting IDM server

Reply:
Ack